tests/unit/array/sort-test.js

```diff
@@ -0,0 +1,83 @@
+import { sort } from 'ember-awesome-macros/array';
```

Comment:
Would you like me to move the non-duplicate tests (the last three, I believe) into the existing integration test file? (they seemed a bit more like unit tests to me, but not sure how/where you're drawing the line?)

The composibility test was copied basically verbatim from ['length-test`](https://github.com/kellyselden/ember-awesome-macros/blob/master/tests/unit/array/length-test.js#L35), and the other two are testing the specific functionality of this patch (ie they fail without the accompanying code change)